test/OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit/Models/Propagated/STK_EGM2008_100x100_2hr_run.csv

```diff
@@ -0,0 +1,122 @@
+YAM.UTCGregorian,YAM.ElapsedSecs,YAM.EarthICRF.X,YAM.EarthICRF.Y,YAM.EarthICRF.Z,YAM.EarthICRF.VX,YAM.EarthICRF.VY,YAM.EarthICRF.VZ

```

Comment:
From the formatting it looks like this file was generated by GMAT, not STK?

Reply:
It looks like that, but I just made all the formatting the same after generating the runs from STK and GMAT so that I didn't have to change the importing fixture within the unit tests (cause it expects this format)

Reply:
Ah gotcha. Disregard all the comments! I figured it was something like that.